third_party/blink/renderer/platform/wtf/text/text_encoding.cc

```diff
@@ -62,8 +62,28 @@ std::string TextEncoding::Encode(const String& string,
   if (string.empty())
     return std::string();
 
-  recordreplay::Assert("[RUN-1350-1765] TextEncoding::Encode %u %s %d", string.length(), GetName(),
-                       (int)handling);
+  // For RUN-1350-1797 (add text contents to assertion.)
+  char buf1[16] = {0};
```

Comment:
I would strongly prefer `#define`'s or `constexpr` here instead of repeating unnamed constants multiple times... even if its temporary code 😶